tool/flutter_ci_script_stable.sh

```diff
@@ -26,6 +26,7 @@ declare -ar PROJECT_NAMES=(
     "ios_app_clip"
     "isolate_example"
     "jsonexample"
+    "material_3_demo"
```

Comment:
Actually, this project requires Dart 2.17.0-0, so we need to comment this entry out with a TODO to add it back in. Please add this project to the matching `tool/flutter_ci_script_beta.sh` config for Beta CI. That way we will get some CI on this code.

Reply:
Got it. Thanks a lot for the suggestion!